README.md

```diff
@@ -412,13 +412,13 @@ less -S k48/HS0674-contigs.sam
 The contig ID is given in the first column, and the position of the contig on the reference is given in the third and fourth columns. Which large contig has two alignments, and what are the positions of these two alignments?
 
 > The contig that has alignments starting at
-> chr3:186,698,393 and chr3:187,439,792.
+> chr3:186,980,605 and chr3:187,722,004.
 
 The orientation is given in the second column. The numbers 0 and 2048 both indicate positive orientation, and 16 indicates negative orientation. What is the position and orientation of these two alignments?
 
 > chr3:186,980,605 (-) and chr3:187,722,004 (+)
 
-In IGV, go to the region `chr3:186,500,000-188,000,000`. The alignment highlighted in blue indicates that the alignment of this contig to the reference genome is split in two, suggesting a misassembly or a structural rearrangement. To find the other half of the alignment, hover over the blue alignment and note its "Read name". Right click on that alignment, select "Select by name", and enter the "Read name" that you noted previously. Both alignments of this one contig are now highlighted in red.
+In IGV, go to the region `chr3:186,500,000-188,000,000`. The alignment highlighted in blue indicates that the alignment of this contig to the reference genome is split in two, suggesting a misassembly or a structural rearrangement. Sometimes, highlighting for split alignments is not enabled by default. Right click on one of the alignments and choose "Link supplementary alignments" to enable highlighting. To find the other half of the alignment, hover over the blue alignment and note its "Read name". Right click on that alignment, select "Select by name", and enter the "Read name" that you noted previously. Both alignments of this one contig are now highlighted in red.
```

Comment:
> Sometimes, highlighting for split alignments is not enabled by default. Right click on one of the alignments and choose "Link supplementary alignments" to enable highlighting.

When you select `Link supplementary alignments`, are these following instructions no longer necessary?

> To find the other half of the alignment, hover over the blue alignment and note its "Read name". Right click on that alignment, select "Select by name", and enter the "Read name" that you noted previously. Both alignments of this one contig are now highlighted in red.

Reply:
I cannot remember very well, but I guess before doing these instructions, all supplementary alignments are in blue, so the connections between pairs are not obvious. Doing this will allow them to see the corresponding pair for a specific alignment. Yeah, it might be unnecessary to mention these details in abyss-activity.

Reply:
So I will wait for your decision; we can skip these details or let them be there.

Reply:
These `Select by name` instructions are no longer needed. Please remove them. The new `Link supplementary alignments` feature is much better!

Please add this text in addition to what you have:

Because the orientations of the two alignments differ, part of the split alignment is highlighted in pink. Hover your cursor over the pink alignment and look at the `Strands` field in the pop up text.

Reply:
Done.